src/app/loadout-builder/LoadoutBuilder.m.scss

```diff
@@ -33,3 +33,26 @@
     min-width: 0;
   }
 }
+
+.processing {
+  position: fixed;
+  display: flex;
+  justify-content: center;
+  right: 0;
+  left: 0;
+  margin: 0 auto;
+  top: 50px;
+  z-index: 50;
+}
+
+.processingContents {
+  background-color: #313233;
+  border-radius: 5px;
```

Comment:
Style wise, black background and no border radius fits better with the overlays we've done elsewhere.

Reply:
I had that originally so I will put it back.

Reply:
How is this
![image](https://user-images.githubusercontent.com/7344652/86451891-de289b00-bd5e-11ea-9867-84ffee1e4fb2.png)